frontend/global/utils.css

```diff
@@ -33,3 +33,20 @@
   white-space: nowrap;
   width: 1px;
 }
+
+.skip-link {
+  position: absolute;
+  top: var(--spacing-unit-s);
+  left: var(--spacing-unit-s);
+  transform: translateY(-9999px);
+  z-index: 9999;
```

Comment:
I'd prefer a `display: none`

Reply:
Just for reference. The reason not to use `display: none` or `visibility: hidden` in this case is that elements hidden like this will be removed from the tab order and screen readers will ignore them – in this case that would defeat the whole purpose of the skip link.